docs/Database.md

```diff
@@ -35,19 +36,23 @@ _TODO: Move to code using http://usejsdoc.org/ format._
 - **resetPasswordExpires** (date)
 
 ## References
-- **_id**
-- **[user](#users)** (_id)
+
+- **\_id**
```

Comment:
`\` before `_id` seems odd?

Reply:
I suppose it has to do with `_` being a special character in markdown.

It seems that both
\_id (`\_id`)
and
_id (`_id`)
work fine with (github) markdown, but the first one removes some ambiguity.

I actually learned here. Didn't know we could escape underscores like that.

Reply:
Funky! At any rate, this doc file is old and outdated and should probably just be removed anyway. 😁

Reply:
Shall we make a separate PR?

Reply:
Yes that's better for separate PR.